django_project/ford3/models/user_management.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import User, Group
+
+
+class ProvinceUserManagementManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(group_id=1)
+
+
+class ProviderUserManagementManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(group_id=3)
+
+
+class CampusUserManagementManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(group_id=2)
+
+
+class UserManagement(models.Model):
```

Comment:
Why do you need this model?

Reply:
to filter based on group without adding any code elsewhere.
do you have any other way to do it?

Reply:
Ok I see but why creating a new table?
There must be another way

Reply:
i don't understand what you meant.
i don't think this will create a new table.
it just query based on group on the existing table.